### PR TITLE
adding documentation for edX exam management commands

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
none

#### What's this PR do?
adding documentation for edX exam management commands

#### How should this be manually tested?
read the file

### CODE REVIEW

File: exams/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Edx  Proctored Exam
+
+### How to populate the edx exam coupons
+
+This management command takes one argument, the csv file. Each file contains coupon codes for one
```
Comment (pdpinch) [inline exams/README.md:5]:
```suggestion
This management command takes one argument, the csv file as provided by edX. Each file contains coupon codes for one 
```

File: exams/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+extract the course number, and the edx_course_key of the exam-course on edX. Make sure that the 
+`course_number` field is set on the course associated with the exam.
+
+Run the command:
+    
+    python manage.py populate_edx_exam_coupons <file_name.csv>
+
+
+## Import exam grades
+This management command users the course_id in each row to find the course and corresponding ExamRun
```
Comment (pdpinch) [inline exams/README.md:16]: typo

```suggestion
This management command uses the course_id in each row to find the course and corresponding ExamRun 
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
Thanks, this is great. 

Just one clarification and a one typo fix.

**Comment by annagav** [comment]
@pdpinch is this good to merge?

**Review by pdpinch — APPROVED** [review]
👍

**Pull request merged**